PowerEditor/installer/nativeLang/english.xml

```diff
@@ -970,19 +970,19 @@ You can define several column markers by using white space to separate the diffe
                 <Item id="7004" name="Sort &amp;tabs"/>
             </Window>
             <ColumnEditor title="Column Editor">
-                <Item id="2023" name="Text to Insert"/>
-                <Item id="2033" name="Number to Insert"/>
-                <Item id="2030" name="Initial number :"/>
-                <Item id="2031" name="Increase by :"/>
-                <Item id="2035" name="Leading zeros"/>
-                <Item id="2036" name="Repeat :"/>
-                <Item id="2032" name="Format"/>
-                <Item id="2024" name="Dec"/>
-                <Item id="2025" name="Oct"/>
-                <Item id="2026" name="Hex"/>
-                <Item id="2027" name="Bin"/>
-                <Item id="1" name="OK"/>
-                <Item id="2" name="Cancel"/>
```

Comment:
OK & Cancel entries should not be modified as well in english.xml

Reply:
Agreed.
Sorry, I changed them in `nativeLang.xml` for testing but forgot `english.xml`.
:-(